android/build.gradle

```diff
@@ -25,8 +25,8 @@ android {
     defaultConfig {
         minSdkVersion 22
         targetSdkVersion 29
-        versionCode 48
-        versionName "v1.10.2-36fe8addc39-g74a18b3359b"
+        versionCode 49
```

Comment:
Do you intend to publish a 1.11 unstable version? If not, bumping the version code is not necessary.

Reply:
I'm planning to use the Open testing track in the Play Store to deliver 1.11.x builds, like those done for other platforms in https://pkgs.tailscale.com/unstable/

We'd link to it on Twitter to find a few users willing to try early builds.